rerun_py/tests/unit/test_arrows3d.py

```diff
@@ -0,0 +1,240 @@
+from __future__ import annotations
+
+import itertools
+from typing import Optional, cast
+
+import numpy as np
+import rerun.experimental as rr2
+from rerun.experimental import cmp as rrc
+from rerun.experimental import dt as rrd
+
+U64_MAX_MINUS_1 = 2**64 - 2
+U64_MAX = 2**64 - 1
+
+
+def test_arrows3d() -> None:
+    arrows_arrays: list[rrd.Arrow3DArrayLike] = [
+        [],
+        # Arrow3DArrayLike: Sequence[Arrow3DLike]: Arrow3D
+        [
+            rrd.Arrow3D(origin=[1, 2, 3], vector=[4, 5, 6]),
+            rrd.Arrow3D(origin=[10, 20, 30], vector=[40, 50, 60]),
+        ],
+    ]
+
+    radii_arrays: list[rrc.RadiusArrayLike | None] = [
+        None,
+        [],
+        np.array([]),
+        # RadiusArrayLike: Sequence[RadiusLike]: float
+        [1, 10],
+        # RadiusArrayLike: Sequence[RadiusLike]: Radius
+        [
+            rrc.Radius(1),
+            rrc.Radius(10),
+        ],
+        # RadiusArrayLike: npt.NDArray[np.float32]
+        np.array([1, 10], dtype=np.float32),
+    ]
+
+    colors_arrays: list[rrc.ColorArrayLike | None] = [
```

Comment:
we're copy pasting this thing into everything that uses colors? Seems a bit of a waste of space for this huge list of variations. I don't mind a bit of copy & paste in those tests, but feels like this is gonna get out of hand. Mind creating a reusable utilities for reoccurring, large component arrays like this?